features/providers/check_pending_employment.feature

```diff
@@ -0,0 +1,97 @@
+Feature: check_pending_employment
```

Comment:
Consider plain english description with more context for persons less aware of business logic

Reply:
Removed underscores and added capital C